master/docs/relnotes/index.rst

```diff
@@ -10,6 +10,84 @@ Release Notes
 
 .. towncrier release notes start
 
+Buildbot ``2.5.2-dev85`` ( ``2019-12-26`` )
```

Comment:
Don't commit updated release notes, they will be generated on release. We should perhaps remove this functionality from `make docs` to prevent confusion.

Reply:
Thank you for the tip

Reply:
it is still in the PR. please remove it!